### PR TITLE
PASS1-89: Show exported filename when exporting wallet to microSD

### PR DESCRIPTION
**Pull request by coreylakey**

### CODE REVIEW

File: ports/stm32/boards/Passport/modules/new_wallet.py [hunk 1]
```diff
@@ -581,7 +587,13 @@ async def show(self):
                 self.exported = True
                 self.save_new_wallet_progress()
 
-                dis.fullscreen('Saved to microSD')
+                # dis.fullscreen('Saved to microSD')
```
Comment (FoundationKen) [inline ports/stm32/boards/Passport/modules/new_wallet.py:590]: Delete this line

File: ports/stm32/boards/Passport/modules/new_wallet.py [hunk 2]
```diff
@@ -547,18 +547,24 @@ async def show(self):
...
                     with CardSlot() as card:
                         # Make a filename with the option of injecting the sd path, hash of the data, acct num, random number
-                        fname = filename_pattern.format(sd=card.get_sd_root(), hash=data_hash, acct=self.acct_num, random=random_hex(8))
-                        # print('Saving to fname={}'.format(fname))
+                        fname = filename_pattern.format(sd=card.get_sd_root(),
+                                                        hash=data_hash,
+                                                        acct=self.acct_num,
+                                                        random=random_hex(8),
+                                                        xfp=xfp2str(settings.get('xfp')).lower())
+                        print('Saving to fname={}'.format(fname))
```
Comment (FoundationKen) [inline ports/stm32/boards/Passport/modules/new_wallet.py:567]: Comment out this line again

File: ports/stm32/boards/Passport/modules/new_wallet.py (hunk 1 above)
Comment (coreylakey) [reply ports/stm32/boards/Passport/modules/new_wallet.py]: Done.

File: ports/stm32/boards/Passport/modules/new_wallet.py (hunk 2 above)
Comment (coreylakey) [reply ports/stm32/boards/Passport/modules/new_wallet.py]: Done.

**Review by FoundationKen — APPROVED** [review]

**Pull request merged**